README.md

```diff
@@ -138,6 +138,15 @@ If you have any questions or discussion topics, start a [GitHub Discussion][8].
 ## Contributing
 Contributions are always welcome. Please read [Contributing.md](CONTRIBUTING.md) if you are considering making contributions.
 
+## Related projects
+- [i3-resurrect](https://github.com/JonnyHaystack/i3-resurrect) - "Simple solution to saving and restoring i3 workspaces"
+- [tmux-resurrect](https://github.com/tmux-plugins/tmux-resurrect)
+- [tmux-continuum](https://github.com/tmux-plugins/tmux-continuum) - an excellent companion to
```

Comment:
This can be removed as it doesn't have anything to do with this script, just tmux-resurrect.